src/main/java/seedu/address/logic/parser/DeleteModuleCommandParser.java

```diff
@@ -16,6 +16,7 @@
      *
      * @param args args for deleting a module
      * @throws ParseException if {@code userInput} does not conform the expected format
+     * @return
```

Comment:
what's returned? for the following changes also

Reply:
Thanks for pointing this out!!